src/main/java/com/atlauncher/gui/tabs/settings/GeneralSettingsTab.java

```diff
@@ -238,6 +240,38 @@ public GeneralSettingsTab() {
         }
         add(enableDiscordIntegration, gbc);
 
+        // Enable Feral Gamemode
+        if (OS.isLinux()) {
+          boolean gameModeExistsInPath = Utils.executableInPath("gamemoderun");
+
+          gbc.gridx = 0;
+          gbc.gridy++;
+          gbc.insets = LABEL_INSETS;
+          gbc.anchor = GridBagConstraints.BASELINE_TRAILING;
+          enableFeralGamemodeLabel = new JLabelWithHover(GetText.tr("Enable Feral Gamemode") + "?", HELP_ICON,
+                  GetText.tr("This will enable Feral Gamemode for packs launched."));
+          add(enableFeralGamemodeLabel, gbc);
+
+          gbc.gridx++;
+          gbc.insets = FIELD_INSETS;
+          gbc.anchor = GridBagConstraints.BASELINE_LEADING;
+          enableFeralGamemode = new JCheckBox();
+          if (App.settings.enableFeralGamemode()) {
+              enableFeralGamemode.setSelected(true);
+          }
+
+          if (! gameModeExistsInPath) {
+            enableFeralGamemodeLabel.setToolTipText("This will enable Feral Gamemode for packs launched (disabled because gamemoderun not found in PATH, please install Feral Gamemode or add it to your PATH).");
```

Comment:
This text should also be wrapped in `GetText.tr()`

Reply:
Done :smile: